docs/ilm.asciidoc

```diff
@@ -0,0 +1,190 @@
+This prevents indices from being created before policies are changed, and allows policies to be applied to indices at creation time.
+It also excludes setup from the ingestion process, which allows you to ensure ILM is setup correctly before using APM.
+
+Index lifecycle management will manage an index based on its defined policy.
+Run the <<setup-command>> with the ` --index-management` flag to set up the default ILM policy:
+
+[source,js]
+-----------------------
+./apm-server setup --index-management
+-----------------------
+// CONSOLE
+
+//Policies only need to be created once, and will persist through version upgrades.
+You can confirm the policy was created with the GET lifecycle policy API.
+Here's what the transaction response looks like:
+
+[source,js]
+-----------------------
+GET _ilm/policy
+{
+    "apm-7.2.0-transaction": {
+        "version": 1,
+        "modified_date": "2019-05-28T15:55:26.791Z",
+        "policy": {
+            "phases": {
+                "warm": {
+                    "min_age": "31d",<3>
+                    "actions": {
+                        "readonly": {},
+                        "set_priority": {
+                            "priority": 50<4>
+                        }
+                    }
+                },
+                "hot": {
+                    "min_age": "0ms",
+                    "actions": {
+                        "rollover": {
+                            "max_size": "50gb",<1>
+                            "max_age": "7d"<2>
+                        },
+                        "set_priority": {
+                            "priority": 100<4>
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+-----------------------
+// CONSOLE
+<1> Rollover after _50gb_
+<2> Rollover after _7 days_
+<3> Move to warm phase after _31 days_
+<4> Priority for recovering your indices after a node restart. Higher priorities are recovered first.
+
+[float]
+[[ilm-edit]]
+=== Editing ILM defaults
+
+Once you've enabled index lifecycle management and setup the ILM policies, you can view and edit the policies in the Index lifecycle policies UI in Kibana. For more information about working with the UI,
+see {kibana-ref}/index-lifecycle-policies.html[*Index lifecycle policies*].
+
+Index policies and templates are prefixed with version numbers, for example: `apm-7.2.0-metric-000001`.
+This means if you use APM Server, modify the policy, and upgrade, you'll need to do two things:
+1. Set `ilm.enabled` to false and configure your indices to the old ILM indices in the yml file.
+2. Update the policies manually when you upgrade. When APM Server is started for the first time,
+APM Server checks to see if the policies are there for this version, if they're not, it creates them.
+
+
+// NOTES BELOW...
+// **************
+
+// True?? If you do not run the setup command, policies will will not be applied immediately??
```

Comment:
false, they should always get applied. The setup cmd is just a cleaner separation between setting everything up and ingesting data.